test/modules/backup-azure/backup_backend_test.go

```diff
@@ -34,7 +33,23 @@ import (
 	"github.com/weaviate/weaviate/usecases/config"
 )
 
-func Test_AzureBackend_Backup(t *testing.T) {
+var override = []string{"", ""}
+
+func Test_AzureBackend_Start(t *testing.T) {
+	overrides := [][]string{
+		{"", ""},
+		{"testbucketoverride", "testBucketPathOverride"},
+	}
+
+	override = overrides[0]
+	azureBackend_Backup(t)
```

Comment:
Can those methods be enhanced with method parameters? so that we could remove this global `override` array, it would add a lot of visibility to those tests.

Also can we refactor this method and run those tests but with different test cases like we do it [here](https://github.com/weaviate/weaviate/blob/main/test/modules/text2vec-aws/text2vec_aws_test.go#L31-L52) for example?

Reply:
done

Reply:
overrides removed